pkg/server/server.go

```diff
@@ -54,8 +57,41 @@ func (p PizzaOvenServer) Run(serverPort string) {
 }
 
 type reqData struct {
-	URL  string `json:"url"`
-	Wait bool   `json:"wait,omitempty"`
+	URL      string `json:"url,omitempty"`
+	Wait     bool   `json:"wait,omitempty"`
+	Org      string `json:"org,omitempty"`
+	Archives bool   `json:"archives,omitempty"`
+}
```

Comment:
I wonder if a different endpoint makes sense. Maybe `/bake-org`. The way this is written, there's alot of weird stuff that happen. What if a user provides both a `URL` and an `Org`? What if they have `URL` and `Archieves` set? From the data perspective, it sort of mixes too many concerns for a single payload type.